walletlibrary/src/main/java/com/microsoft/walletlibrary/networking/formatters/OpenId4VciIssuanceRequestFormatter.kt

```diff
@@ -53,15 +55,17 @@ internal class OpenId4VciIssuanceRequestFormatter(private val libraryConfigurati
         return signContents(claims, identifier)
     }
 
-    private fun signContents(contents: OpenID4VCIJWTProofClaims, responder: Identifier): String {
+    private fun signContents(contents: OpenID4VCIJWTProofClaims, holderIdentifier: HolderIdentifier): String {
         val serializedResponseContent = libraryConfiguration.serializer.encodeToString(
             OpenID4VCIJWTProofClaims.serializer(),
             contents
         )
-        return libraryConfiguration.tokenSigner.signWithIdentifier(
-            serializedResponseContent,
-            responder,
-            com.microsoft.walletlibrary.util.Constants.OPENID4VCI_TYPE_HEADER
-        )
+        formatJwsHeaderForIdentifier(holderIdentifier)
+        return holderIdentifier.sign(serializedResponseContent)
+    }
+
+    private fun formatJwsHeaderForIdentifier(holderIdentifier: HolderIdentifier) {
+        val jwsHeader = JwsHeaderFormatter.formatHeader(holderIdentifier, com.microsoft.walletlibrary.util.Constants.OPENID4VCI_TYPE_HEADER)
```

Comment:
I would want a double-check on this but isn't this type value here the `typ` header, which we've always had `JWT`?

Reply:
We use a different typ header for OIDC4VCI protocol. We use "JWT" for other flows.